encoding/gocode/binding.tmpl

```diff
@@ -5,7 +5,8 @@ import (
 	"io/fs"
 	"path/filepath"
 
-    "cuelang.org/go/cue/build"
+	{{ if .CUEPath }}"cuelang.org/go/cue"{{ end }}
+	"cuelang.org/go/cue/build"
```

Comment:
shouldn't be necessary? we have a call to goimports in the generator that eliminates unused imports if they're not needed

not a big deal, though

Reply:
Well, it wasn't _necessary_, but this is the message thema emits without that line:

```goimports added the following import statements to dashboard_binding_gen.go:
	"cuelang.org/go/cue"
Relying on goimports to find imports significantly slows down code generation. Either add these imports with an AST manipulation in cfg.ApplyFuncs, or set cfg.IgnoreDiscoveredImports to true```